### PR TITLE
Maint/master/re 3611

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -194,4 +206,65 @@ end
...
+    break unless vm_ip == nil
+    sleep 60
+  end
+end
+
+def ship_vm_to_dir(vmpath, destination)
+  FileUtils.cp(vmpath, destination)
+end
+
+def ex(command)
```
Comment (melissa) [inline Rakefile:268]: Ahh, the `ex` command <3

File: Rakefile [hunk 2]
```diff
@@ -125,6 +127,16 @@ task :student do
   Rake::Task["post"].execute
 end
 
+## This requires some configs 
+desc "Package and ship a VM"
+task :ship do
+  vmname = ENV['VMNAME'].split(".").first
+  ovapath=retrieve_ova(vmname)
+  push_vmware(ovapath)
+  ship_vm_to_dir(ovapath, "/mnt/nfs/EducationVMS/#{ENV['VMTYPE']}")
```
Comment (melissa) [inline Rakefile]: So this is supposed to be run on a specific machine, yes? So that it ends up in the correct place when you ship the ova?

Comment (shrug) [inline Rakefile]: Right now it's tied to pe-vm-builder-1, but yeah it would have to be tied to a set of builders that had that nfs export mounted.

We can add other ship methods later (rsync, s3, whatevs) ass needed

**Comment by melissa** [comment]
Hmm, I'm not seeing anything that sticks out in particular. It's hard to follow since I have such little experience with this.

I do see where you mentioned declaring vars multiple times in different places. Why is that again?
